node_client/pyXain/pyXain/pyXain.py

```diff
@@ -384,7 +407,14 @@ def submit_comment(self, author_id, paper_id, comment_directory):
         # large directory. More recent versions have this, but I haven't tried building
         # them from source to try out. This should be remedied in the future.
         try :
-            reply = self.api.add(version_path, recursive=True)
+            # go-ipfs api v0.4.7 has a bug that prevents recursively adding directories.
```

Comment:
Why does this comment+functionality occur in two places? Can it be made into its own function?

Reply:
There is a lot of code redundancy in the submit_manuscript, submit_revision, and submit_comment that I want to eliminate in a future PR. I could add that to this one though as it is probably something I'll work on next.

Reply:
OK